projects/packages/publicize/src/rest-api/class-scheduled-actions-controller.php

```diff
@@ -293,15 +329,15 @@ public function create_item_permissions_check( $request ) {// phpcs:ignore Varia
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function create_item( $request ) {
-		$post_id = $request['postId'];
 
 		if ( Utils::is_wpcom() ) {
 			require_lib( 'publicize/class.publicize-actions' );
 
+			$post_id       = $request['post_id'];
 			$blog_id       = get_current_blog_id();
 			$user_id       = get_current_user_id();
 			$connection_id = (int) $request['connection_id'];
-			$message       = sanitize_textarea_field( $request['message'] );
+			$message       = sanitize_textarea_field( $request['message'] ?? '' );
```

Comment:
Why do we need this change?

Reply:
Because that `message` argument is not required.